### PR TITLE
Website icon colour change from teal to purple

### PR DESCRIPTION
**Pull request by parisa-0**
## Description
 

Changed the svg file for website icon since it was different to other svg files and used fill icon_colour instead for best practice

https://github.com/WomenCodingCommunity/WomenCodingCommunity.github.io/issues/101

## Change Type
- [X] Bug Fix
- [ ] New Feature
- [ ] Code Refactor
- [ ] Mentor Update
- [ ] Documentation
- [ ] Other

## Related Issue

https://github.com/WomenCodingCommunity/WomenCodingCommunity.github.io/issues/101

## Screenshots

Before:
image

After:
image

## Pull request checklist

Please check if your PR fulfills the following requirements:
- [X] I checked and followed the [contributor guide](CONTRIBUTING.md) 
- [X] I have tested my changes locally.
- [X] I have added a screenshot from the website after I tested it locally

### CODE REVIEW

File: program-book-club.html
Comment (lilyndk) [inline program-book-club.html:1]: @parisa-0 these file changes seem to not belong to this PR/issue?

File: _includes/icons/website.svg
Comment (lilyndk) [inline _includes/icons/website.svg:1]: As discussed with @parisa-0 , the hardcoded color variables should be replaced with fill="icon-color" to follow the same structure as the other working networking icons and reuse the icon-color variable.

But I see that you've also moved the existing code several lines down @parisa-0 , which is not necessary :)

File: program-book-club.html
Comment (lilyndk) [reply program-book-club.html:1]: but I guess it's fine if you've discussed it with others to include this, it's just not clear from the related issue #101 or the PR description.

Comment (parisa-0) [reply program-book-club.html:1]: Yes my changes from the another branch have merged with this change, I will need to fix this

Comment (dricazenck) [reply program-book-club.html:1]: Please share with us again when is ready for review.
